app/templates/settings/integrations.hbs

```diff
@@ -0,0 +1,15 @@
+{{title "Settings"}}
+
+<div class="container">
+  <div class="page-header">
+    <h3>Your account integrations</h3>
+  </div>
+
+  <div class="settings">
+    {{#if model.githubAuthToken}}
+      Your account is already connected to github.
+    {{else}}
+      {{github-connect}}
+    {{/if}}
```

Comment:
Generally, we may want to handle this via a component. Some sort of "integration card" type of thing.